tests/bench/locomo_baseline.py

```
@@ -0,0 +1,534 @@
+            args.sample_id,
+            args.question_count,
+        )
+
+        print_summary(results, total_elapsed)
+
+        # Generate JSON output
+        if args.json_output:
+            output_file = args.json_output
+        else:
+            output_file = Path(
+                f"tests/bench/eval_results/locomo_baseline_results_{datetime.now().strftime('%Y%m%d_%H%M%S')}.json"
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)